fineract-provider/src/main/java/org/apache/fineract/infrastructure/jobs/api/SchedulerJobApiResource.java

```diff
@@ -74,20 +77,23 @@ public class SchedulerJobApiResource {
     private final ToApiJsonSerializer<JobDetailHistoryData> jobHistoryToApiJsonSerializer;
     private final PortfolioCommandSourceWritePlatformService commandsSourceWritePlatformService;
     private final PlatformSecurityContext context;
+    private final FineractProperties fineractProperties;
 
     @Autowired
```

Comment:
Let's use Lombok to generate the constructor (RequiredArgsConstructor).

Reply:
Done!